docker-compose-dev.yml

```diff
@@ -52,88 +40,111 @@ services:
       - elevate_net
     logging:
       driver: none
-  # redis_bullmq:
-  #   image: 'redis:7.0.0'
-  #   container_name: redis_bullmq
-  #   restart: 'always'
-  #   command: redis-server --appendonly yes --maxmemory-policy noeviction
-  #   ports:
-  #     - '6380:6379'
-  #   networks:
-  #     - elevate_net
-  #   volumes:
-  #     - redis_data:/data
 
   mentoring:
     build: './'
-    image: elevate/mentoring:1.0
+    image: elevate/mentoring:2.4
     volumes:
       - ./src/:/var/src
     ports:
       - '3000:3000'
-    command: ['nodemon', 'app.js']
+    command: ['nodemon', '--trace-warnings', 'app.js']
     environment:
-      - MONGODB_URL=mongodb://mongo:27017/elevate-mentoring
       - KAFKA_URL=kafka:9092
-      - USER_SERIVCE_HOST=http://user:3001
+      - USER_SERVICE_HOST=http://user:3001
+      - REDIS_HOST=redis://redis:6379
+      - SCHEDULER_SERVICE_HOST=http://scheduler:4000
       - DEV_DATABASE_URL=postgres://postgres:postgres@citus_master:5432/mentoring-local
+    deploy:
+      resources:
+        limits:
+          cpus: '4.0' # Set CPU limit to 4 cores
+          memory: '200M' # Set memory limit to 2 GB
     depends_on:
       - kafka
-      - mongo
     networks:
       - elevate_net
   user:
     build: '../user/'
-    image: elevate/user:1.0
+    image: elevate/user:2.4
     volumes:
       - ../user/src/:/var/src
     ports:
       - '3001:3001'
-    command: ['nodemon', 'app.js']
+    command: ['nodemon', '--trace-warnings', 'app.js']
     environment:
-      - MONGODB_URL=mongodb://mongo:27017/elevate-users
       - KAFKA_URL=kafka:9092
       - REDIS_HOST=redis://redis:6379
       - DEV_DATABASE_URL=postgres://postgres:postgres@citus_master:5432/user-local
     depends_on:
       - kafka
-      - mongo
       - redis
     networks:
       - elevate_net
+    deploy:
+      resources:
+        limits:
+          cpus: '4.0' # Set CPU limit to 4 cores
+          memory: '200M' # Set memory limit to 2 GB
   notification:
     build: '../notification/'
-    image: elevate/notification:1.0
+    image: elevate/notification:2.4
```

Comment:
@nevil-mathew its refer to local image

Reply:
It's not the main docker compose file, it's untouched.
This docker compose is intended to be used for development.

Reply:
This is not the main docker compose file, it is to be used for local development